.happo.js

```diff
@@ -0,0 +1,12 @@
+// .happo.js
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+  apiKey: process.env.HAPPO_API_KEY,
+  apiSecret: process.env.HAPPO_API_SECRET,
+  targets: {
+    chrome: new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
```

Comment:
nice, here is my trick for setting the viewport style (which we can show as a tip on the slide)  https://github.com/bahmutov/cypress-react-unit-test/blob/master/examples/visual-testing-with-happo/.happo.js#L4

Reply:
Oh, neat! That's definitely useful -- I also plan on showing how things look like on smaller screens + other browsers.